CHANGELOG.md

```diff
@@ -1,5 +1,104 @@
+* **components:** add notification and message count from array length ([#893](https://github.com/akveo/ng2-admin/issues/893)) ([a8fe3f1](https://github.com/akveo/ng2-admin/commit/a8fe3f1))
+* **dependencies:** fix [#689](https://github.com/akveo/ng2-admin/issues/689), [#688](https://github.com/akveo/ng2-admin/issues/688) ([d34e6e9](https://github.com/akveo/ng2-admin/commit/d34e6e9))
+* **dependencies:** fix angular2-template-loader version ([2b25258](https://github.com/akveo/ng2-admin/commit/2b25258))
+* **dependencies:** fix versions ([62810ae](https://github.com/akveo/ng2-admin/commit/62810ae))
+* **dependencies:** fix versions ([5523029](https://github.com/akveo/ng2-admin/commit/5523029))
+* **dependencies:** update dependencies, fix [#694](https://github.com/akveo/ng2-admin/issues/694), fix [#695](https://github.com/akveo/ng2-admin/issues/695) ([08de1f6](https://github.com/akveo/ng2-admin/commit/08de1f6))
+* **docker:** fix [#712](https://github.com/akveo/ng2-admin/issues/712) ([cd7520e](https://github.com/akveo/ng2-admin/commit/cd7520e))
+* **documents:** fix [#771](https://github.com/akveo/ng2-admin/issues/771) ([78c70ae](https://github.com/akveo/ng2-admin/commit/78c70ae))
+* **editors:** fix an issue where ckeditor doesn't load on the demo ([0a4f40c](https://github.com/akveo/ng2-admin/commit/0a4f40c))
+* **forms:** bad linking correction between label and input ([#886](https://github.com/akveo/ng2-admin/issues/886)) ([bb6ecd3](https://github.com/akveo/ng2-admin/commit/bb6ecd3))
+* **maps:** fix [#813](https://github.com/akveo/ng2-admin/issues/813) ([b414eaf](https://github.com/akveo/ng2-admin/commit/b414eaf))
+* **menu:** fix inactive menu item ([2468710](https://github.com/akveo/ng2-admin/commit/2468710))
+* **package:** add the description fields ([0d77147](https://github.com/akveo/ng2-admin/commit/0d77147))
+* **sidebar:** fix [#997](https://github.com/akveo/ng2-admin/issues/997) ([0a3c98e](https://github.com/akveo/ng2-admin/commit/0a3c98e))
+* **style:** fix [#958](https://github.com/akveo/ng2-admin/issues/958) ([#963](https://github.com/akveo/ng2-admin/issues/963)) ([06d1b98](https://github.com/akveo/ng2-admin/commit/06d1b98))
+* **tables:** fix [#1000](https://github.com/akveo/ng2-admin/issues/1000) ([#1001](https://github.com/akveo/ng2-admin/issues/1001)) ([4b98e0e](https://github.com/akveo/ng2-admin/commit/4b98e0e))
+* **tables:** fix [#700](https://github.com/akveo/ng2-admin/issues/700) ([c85eae7](https://github.com/akveo/ng2-admin/commit/c85eae7))
+* **tables:** fix [#704](https://github.com/akveo/ng2-admin/issues/704) ([0fbad7e](https://github.com/akveo/ng2-admin/commit/0fbad7e))
+* **theme:** fix cards background of blur theme ([0528e2e](https://github.com/akveo/ng2-admin/commit/0528e2e))
+* **ui:** fix NgbModal declaration ([49ff10e](https://github.com/akveo/ng2-admin/commit/49ff10e))
+* **validator:** change email regex to: http://emailregex.com/ ([4470607](https://github.com/akveo/ng2-admin/commit/4470607))
```

Comment:
Since Angular v4 Final it already have a email validator, I don't think it's necessary. People can just use `Validators.email`.